pkg/shell/active-pages-modal.jsx

```diff
@@ -57,7 +56,6 @@ export const ActivePagesDialog = ({ frames }) => {
         return result;
     }
 
-    const Dialogs = useDialogs();
```

Comment:
Not having to have a handle to a global seems like a good improvement, but the `result` argument is just too weird for my taste.